Makefile

```diff
@@ -39,11 +39,9 @@ fonts:
 web-resources:
 	mkdir -p build/$(browser)/$(type)/web_accessible_resources
 	cp shared/data/web_accessible_resources/* build/$(browser)/$(type)/web_accessible_resources/
-	for f in build/$(browser)/$(type)/web_accessible_resources/*; do echo '' >> $$f; done
-	cat build/$(browser)/$(type)/web_accessible_resources/* >> build/$(browser)/$(type)/data/surrogates.txt
-	sed -i.bak 's/^\/\///g' build/$(browser)/$(type)/data/surrogates.txt
-	rm build/$(browser)/$(type)/data/surrogates.txt.bak
-	for f in build/$(browser)/$(type)/web_accessible_resources/*; do echo "$$(sed '1d' $$f)" > $$f; done
+	cp shared/tracker-surrogates/surrogates/*.js build/$(browser)/$(type)/web_accessible_resources/
+	basename build/$(browser)/$(type)/web_accessible_resources/* >> build/$(browser)/$(type)/data/surrogates.txt # fills file with names of surrogates
+	sed -i "" -e 's/^/domain.com\//; s/$$/ application\/javascript\n/' build/$(browser)/$(type)/data/surrogates.txt # prefixes each name with "domain.com" and adds "application/javascript" suffix
```

Comment:
So this will output file like this:

```
domain.com/ad_status.js application/javascript

domain.com/adsbygoogle.js application/javascript

domain.com/amzn_ads.js application/javascript

…
```

which is good enough for us and shaves 10kb off the extension zip file.

BTW I'm adding 'domain.com' for each of those because the domain doesn't matter and surrogate files in this new repo don't have those comments anymore.